hw/top_earlgrey/dv/env/chip_env.core

```diff
@@ -7,84 +7,7 @@ description: "CHIP DV UVM environmnt"
-      - seq_lib/chip_jtag_base_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_alert_handler_escalation_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_full_aon_reset_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_deep_power_glitch_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_main_power_glitch_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_sysrst_ctrl_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_random_sleep_all_reset_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_deep_sleep_all_reset_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_pwm_pulses_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_uart_tx_rx_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_uart_rand_baudrate_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_sysrst_ctrl_inputs_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_sysrst_ctrl_reset_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_sysrst_ctrl_outputs_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_gpio_smoke_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_gpio_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_flash_ctrl_lc_rw_en_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_flash_init_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_flash_rma_unlocked_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_lc_ctrl_transition_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_lc_walkthrough_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_lc_walkthrough_testunlocks_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_lc_ctrl_program_error_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_spi_tx_rx_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_rom_ctrl_integrity_check_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_sram_ctrl_execution_main_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_sram_ctrl_scrambled_access_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_keymgr_key_derivation_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_ast_clk_outputs_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_sensor_ctrl_status_intr_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_pwrmgr_deep_sleep_all_wake_ups_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_adc_ctrl_sleep_debug_cable_wakeup_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_repeat_reset_wkup_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_sw_rstmgr_alert_info_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_rv_dm_ndm_reset_vseq.sv: {is_include_file: true}
-      - seq_lib/chip_callback_vseq.sv: {is_include_file: true}
-      - autogen/chip_env_pkg__params.sv: {is_include_file: true}
-      - ast_ext_clk_if.sv
-      - ast_supply_if.sv
-      - pwrmgr_low_power_if.sv
-    file_type: systemVerilogSource
+      - lowrisc:dv:chip_env_src
 
 generate:
   ral:
```

Comment:
The parts below have to be repeated with additional arguments in the closed-source core file.

Reply:
I am not sure if this segregation is needed. The `filesets` below selectively enables the addition of chip level RAL generated using our open source vanilla command to the generated filelist:
```
    generate:
      - "!skip_ral_gen ? (ral)"
```

In `partner_chip_sim_cfg.hjson`, we could set  the fusesoc flag `skip_ral_gen` using `sv_flist_gen_opts` list, and the `partner_chip_env.core` could specify its own RAL generator.

Reply:
Hm good idea. Let me try that.